### ISSUE TITLE
[question] Will this be merged into the official ncclient repo?

### ISSUE THREAD

**Issue by hyberdk** [CiscoDevNet/ncclient#7]
Hi @einarnn, 

I was looking at this to do a dial-in streaming telemetry of our XE routers. It seems like this repo was never merged into the official ncclient repo, is that still the plan?

I really like the idea of using ncclient to do a dial-in subscription instead of configuring all routers/switches with a dial-out option. 

Esben

**Comment by einarnn** [comment]
@hyberdk, I've just not had the time to get this code merged into the main ncclient repo in a way that maintains backwards compatibility. If there is interest I'm happy to spend some time on it. IIRC, the gNMI interfaces also support dial-in telemetry, so that should also be an option if your need is urgent.

I'll leave this issue open until I've had time to merge the code to the main ncclient repo.

;ch

**Comment by hyberdk** [comment]
Hi @einarnn,

thanks for your reply and sorry about the late response. Since we plan on using netconf for the rest of management it would be nice to "keep it in the same family"..

Right now we are using the dail-out option, but I dont really want to maintain the configs for every telemetry option we want to use. Also the current collectors (both telegraf and pmacct/mdt-diailout-collector) does not really do what we want. I we have primary been testing with telegraf, but that is really married with InfluxDB and that format does not work too well for us as we are using [ClickHouse](https://clickhouse.com/) (which is blazingly fast) as database for our timeseries data.

for now we use telegraf and sending the events to kafka, but it seems like telegraf is "splitting" the events from the producer into multiple events to support the Influx format which I was hoping to avoid as it seems to give some duplicate data.. 

Long story short, I will test your branch out and see if it works like we like so I am not in a big hurry, I just didnt want to build a new client based on something that is not going to be maintained in the future. Im an not in a big hurry as we are just dipping our feet in the MDT and can workaround the shortcomings with telegraf for now.

Esben

**Comment by einarnn** [comment]
Ok-dokey. Let me know if there is anything else.
